api.go

```diff
@@ -23,6 +28,24 @@ type carAvailabilityResp struct {
 	Longitude   float64 `yaml:"Longitude"`
 }
 
+type flexCarAvailabilitiesResp struct {
+	Data flexVehiclesResp `json:"d"`
+}
+
+type flexVehiclesResp struct {
+	Vehicles []FlexCarAvailabilityResp `json:"Vehicles"`
+}
+
+type FlexCarAvailabilityResp struct {
```

Comment:
This shouldn't be exported 
```suggestion
type flexCarAvailabilityResp struct {
```